### PR TITLE
INC-544: behaviour entry graph

### PR DESCRIPTION
**Pull request by neilmendum**

### CODE REVIEW

**Comment by ushkarev** [comment]
"server/views/partials/analytics/chartBasePercentageBars.njk" still uses an old colour class `app-chart-colour--a`

File: server/routes/analyticsRouter.ts [hunk 1]
```diff
@@ -63,13 +63,15 @@ const protectedCharacteristicsChartContent: Record<string, CharacteristicsChartC
       'This chart will help you tell if there is any imbalance in incentive levels between prisoners with a disability recorded on NOMIS and those without. As the amount of missing data is high, the number of prisoners with unknown information is also provided.',
     labelColumn: 'Disability',
     graphGoogleAnalyticsCategory: 'Incentive level by disability',
+    wideLabel: true,
```
Comment (ushkarev) [inline server/routes/analyticsRouter.ts:66]: i added a type to describe this object in the other PR so you'll now need to put `wideLabel?: boolean` in there now

Comment (xoen) [reply server/routes/analyticsRouter.ts:67]: Yep yep, test failed after we rebased. I just sorted this out 👌

**Comment by xoen** [comment]
> "server/views/partials/analytics/chartBasePercentageBars.njk" still uses an old colour class app-chart-colour--a
Well spotted! Sorted here: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/163/commits/ac5a78ffa7082d463bddb21b3240a849b7b2456c

**Review by ushkarev — APPROVED** [review]

**Pull request merged**